Build/set-dependencies.sh

```diff
@@ -115,5 +115,6 @@ for PACKAGE in Neos.Eel Neos.FluidAdaptor Neos.Kickstarter ; do
 done
 
 php "${COMPOSER_PHAR}" --working-dir=Packages/Framework/Neos.Kickstarter require --no-update "neos/fluid-adaptor:~${BRANCH}.0"
+php "${COMPOSER_PHAR}" --working-dir=Packages/Application/Neos.Welcome require --no-update "neos/flow:~${BRANCH}.0"
```

Comment:
In any case this needs to be amended with a push in the package, otherwise that change will never make it upstream…

Reply:
Doesn't the create-branch script do that after invoking set-dependencies? https://github.com/neos/flow-development-distribution/blob/master/Build/create-branch.sh#L50

Reply:
🤔 TBH I just checked the repo and… https://github.com/neos/flow-welcome/tree/7.0 is not where it should be, then.

Reply:
Indeed, Welcome is pushed, my bad!